test/e2e/v2/e2e_suite.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("kubebuilder", func() {
 			err := kbc.Init(
 				"--project-version", "2",
 				"--domain", kbc.Domain,
-				"--dep=false")
+				"--fetch-deps=false")
```

Comment:
Does this imply that there was a backwards incompatible change?

Reply:
Yes, these [particular flags](https://github.com/kubernetes-sigs/kubebuilder/blob/c52ba01/cmd/init_project.go#L94-L98) are not replicated in v2 from v1 plugin `BindFlags`. To be backwards-compatible I will add deprecated flags to v2.

Reply:
I actually looked into the flags and this change is ok, you can mark this conversation as resolved to reduce the noise.